bridetalk_project/src/components/molecule/forms/createuserform/CreateUserForm.jsx

```diff
@@ -0,0 +1,71 @@
+import {
+  Headline1,
+  EnterText,
+  EnterEmail,
+  EnterPassword,
+  DropdownLocation,
+  PrimaryButton,
+  CancelButton,
+} from "../../../atoms";
+
+import "./createuserform.css";
+import { Link } from "react-router-dom";
+
+import { useNavigate } from "react-router-dom";
+export const CreateUserForm = () => {
+  let navigate = useNavigate();
+
+  const toFrontPage = () => {
```

Comment:
Very nice work extracting paths in variables and making it like that :)

My comment would be, you have 3 functions that serve exactly the same purpose.
Why not making a generic function instead, that has a path parameter? 
Then you could say f.ex., 

```
navigateTo('/login)
navigateTo(-1)
navigateTo('/)
```
The less repetition the better